.github/workflows/build-test.yml

```diff
@@ -878,10 +878,10 @@ jobs:
         if: needs.pick_environment.outputs.release_channel == 'stable'
         uses: actions/create-release@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN_TO_CREATE_RELEASE }}
         with:
           tag_name: ${{steps.version.outputs.version}}
           release_name: Release ${{steps.version.outputs.version}}
           body_path: RELEASE_CHANGELOG.md
-          draft: true
+          draft: false
```

Comment:
No idea why this was draft, looks like David changed this for some reason (possible trying to figure out why things didn't work).
I'm reverting it.